src/metabase/api/fingerprint.clj

```diff
@@ -0,0 +1,119 @@
+(ns metabase.api.fingerprint
+  (:require [compojure.core :refer [GET POST PUT]]
+            [metabase.api.common :as api]
+            [metabase.fingerprinting :as fingerprinting]
+            (metabase.models [field :refer [Field]]
+                             [table :refer [Table]]
+                             [segment :refer [Segment]]
+                             [card :refer [Card]])
+            [schema.core :as s]))
+
+(def ^:private ^:const MaxQueryCost       (s/maybe (s/enum "cache"
+                                                           "sample"
+                                                           "full-scan"
+                                                           "joins")))
+(def ^:private ^:const MaxComputationCost (s/maybe (s/enum "linear"
+                                                           "unbounded"
+                                                           "yolo")))
+(def ^:private ^:const Resolution         (s/maybe (s/enum "month"
+                                                           "day"
+                                                           "raw")))
```

Comment:
the "block indentation rule" doesn't really apply for var definitions that span multiple lines each. It might make more sense to just do something like

```clojure
(def ^:private ^:const Resolution        
  (s/maybe (s/enum "month"
                   "day"
                   "raw")))
```